src/const.h

```diff
@@ -695,6 +695,9 @@ static constexpr int32_t PSTRG_OUTFITS_RANGE_SIZE = 500;
 static constexpr int32_t PSTRG_MOUNTS_RANGE_START = (PSTRG_RESERVED_RANGE_START + 2001);
 static constexpr int32_t PSTRG_MOUNTS_RANGE_SIZE = 10;
 static constexpr int32_t PSTRG_MOUNTS_CURRENTMOUNT = (PSTRG_MOUNTS_RANGE_START + 10);
+//[4000 - 7000];
+static constexpr uint32_t PSTRG_BESTIARY_RANGE_START = (PSTRG_RESERVED_RANGE_START + 4000);
+static constexpr uint32_t PSTRG_BESTIARY_RANGE_SIZE = 3000;
```

Comment:
Is this enough? How many monsters do we have now? It's been growing fast in Tibia and should probably have space for a few more years at least.

Reply:
> Is this enough? How many monsters do we have now? It's been growing fast in Tibia and should probably have space for a few more years at least.

the last time I generated wiki monsters it generated 1657 files, but there are some that are bosses in different states and monsters with looktypeex looking in different directions. There must be more than 1500 for sure. The CIP averages between 30 and 90 monsters per upgrade.
I think we should have a table to store separately.

Reply:
> Is this enough? How many monsters do we have now? It's been growing fast in Tibia and should probably have space for a few more years at least.

slightly above 700. RaceId is sent as u16 in network messages